bundles/org.dataflowanalysis.pcm.extension.dictionary.characterized/model/DataDictionaryCharacterized.genmodel

```diff
@@ -19,8 +19,8 @@
   <modelPluginVariables>CDO=org.eclipse.emf.cdo</modelPluginVariables>
   <editPluginVariables>CDO_EDIT=org.eclipse.emf.cdo.edit</editPluginVariables>
   <genPackages prefix="DataDictionaryCharacterized" basePackage="org.dataflowanalysis.pcm.extension.dictionary.characterized"
-      disposableProviderFactory="true" extensibleProviderFactory="true" childCreationExtenders="true"
```

Comment:
These changes seem to be unrelated and might be an automatic change from opening the genmodel.
Is there a reason for this change?

Reply:
GitHub Diff is a bit misleading here, I only set `generateExampleClass` to false, as it creates an empty test model project.